### ISSUE TITLE
transpose instruction in V field is ignored

### ISSUE THREAD

**Issue by goplayerjuggler** [paulrosen/abcjs#916]
I used the sample ABC from the standard:
https://abcnotation.com/wiki/abc:standard:v2.1#multiple_voices

X:1
T:Zocharti Loch
C:Louis Lewandowski (1821-1894)
M:C
Q:1/4=76
%%score (T1 T2) (B1 B2)
V:T1           clef=treble-8  name="Tenore I"   snm="T.I"
V:T2           clef=treble-8  name="Tenore II"  snm="T.II"
V:B1  middle=d clef=bass      name="Basso I"    snm="B.I"  transpose=-24
V:B2  middle=d clef=bass      name="Basso II"   snm="B.II" transpose=-24
K:Gm
%            End of header, start of tune body:
% 1
[V:T1]  (B2c2 d2g2)  | f6e2      | (d2c2 d2)e2 | d4 c2z2 |
[V:T2]  (G2A2 B2e2)  | d6c2      | (B2A2 B2)c2 | B4 A2z2 |
[V:B1]       z8      | z2f2 g2a2 | b2z2 z2 e2  | f4 f2z2 |
[V:B2]       x8      |     x8    |      x8     |    x8   |

The dots in the bass clef are too high, it looks like 3 octaves too high. It seems the transpose instruction isn't handled.

**Comment by paulrosen** [comment]
Yes, that's a bug. Thanks for reporting it and I'll try to get to it soon.

**Comment by oliver-daniel** [comment]
Hi, confirming that this is still the case in May 2024. Beggars can't be choosers, but I'd love to see this back on the roadmap!

**Comment by seisiuneer** [comment]
transpose shifts playback pitch, not visual representation

octave shifts playback and visual representation

For single voice tunes this does shifts the visual representation up and playback octave transposed down to the original pitch, but does not work when the same combination is used on the V: tag for multi-voice tunes:

K: Edor octave=1 transpose=-12
